app/controllers/users_controller.rb

```diff
@@ -17,35 +17,35 @@ def create
 
     if @user.save
       @user.send_activation_email
-      flash.now[:success] = t".flash"
+      flash.now[:success] = t ".flash"
       redirect_to @user
     else
-      flash.now[:danger] = t".danger"
+      flash.now[:danger] = t ".danger"
       render :new
     end
   end
 
   def show
     return if @user
```

Comment:
đoạn này không pass qua thằng find_user thì how to nhảy vào đây để mà check nứa?

Reply:
Đoạn này thằng find_user có đk đụng đến create đâu mà cần phải qua nó chị em chỉ cho nó hoạt động trong các method edit show destroy chứ create cần gì phải qua find_user đâu chị

Reply:
Em đã sửa lại rùi chị